setup.py

```diff
@@ -256,9 +256,11 @@ def finalize_options(self):
         Extension('gensim.models.doc2vec_inner',
             sources=['./gensim/models/doc2vec_inner.c'],
             include_dirs=[model_dir]),
+        Extension('gensim.corpora._mmreader',
+            sources=['./gensim/corpora/_mmreader.c']),
```

Comment:
`include_dirs` no needed here?

Reply:
no, I don't think it's needed.  The other pyx/c files include model_dir because they reference each other, but this module doesn't need to reference any of the other cython files.